### DISCUSSION TITLE
How to run G16 and ORCA ?

### DISCUSSION THREAD

**Discussion by johnwkeller54** [cyllab/CalcUS#36] [General]
G16 requires ownership of the /usr/local/g16 directory to be "root:gaussian". Is the CalcUS account a member of the gaussian group?  If not, how is CalcUS able to run a gaussian job?
ORCA 5: calculation is started but system freezes. 

(These issues need to be addressed in the documentation, not the git discussion page.)

**Comment by RaphaelRobidas** [comment]
For Gaussian: The Docker processes are launched as the user who created the CalcUS directory (the UID and GID are set inside the `.env` file). I am not entirely sure, but I assume that CalcUS will be able to run G16 if that user is in the gaussian group. In my case, the G16 directory is owned by my user account, so there is no such issue.

For ORCA 5: the entire system freezes? Maybe the machine is running out of memory. In the `.env` file you shared, `OMP_STACKSIZE` was set to 2048M and 4 cores were selected. Thus, the machine should have more memory than 8GB. ORCA doesn't strictly limit itself to the specified amount of memory, so if you're running short of memory you could reduce the number a bit.

You can try on a tiny molecule for debugging purposes, any message would be helpful.

I agree that this sort of issues should be addressed in the documentation, but unfortunately the low volume of users means that few issues have been reported before. I will update the "Troubleshooting" section of the documentation with solutions to this kind of issues for future users.

**Comment by johnwkeller54** [comment]
Thanks, Raphael. I incorrectly assumed the orca memory was total maximum. I’ll try H2O.John K. Sent from Mail for Windows From: Raphaël RobidasSent: Monday, July 24, 2023 11:46 AMTo: cyllab/CalcUSCc: johnwkeller54; AuthorSubject: Re: [cyllab/CalcUS] How to run G16 and ORCA ? (Discussion #36) For Gaussian: The Docker processes are launched as the user who created the CalcUS directory (the UID and GID are set inside the .env file). I am not entirely sure, but I assume that CalcUS will be able to run G16 if that user is in the gaussian group. In my case, the G16 directory is owned by my user account, so there is no such issue.For ORCA 5: the entire system freezes? Maybe the machine is running out of memory. In the .env file you shared, OMP_STACKSIZE was set to 2048M and 4 cores were selected. Thus, the machine should have more memory than 8GB. ORCA doesn't strictly limit itself to the specified amount of memory, so if you're running short of memory you could reduce the number a bit.You can try on a tiny molecule for debugging purposes, any message would be helpful.I agree that this sort of issues should be addressed in the documentation, but unfortunately the low volume of users means that few issues have been reported before. I will update the "Troubleshooting" section of the documentation with solutions to this kind of issues for future users.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by johnwkeller54** [comment]
Ok, I moved the /g16 directory into /home/user and did $ chown -R
user:gaussian .  Now it works.
